Client/Telemetry/CorePing.swift

```diff
@@ -94,6 +94,13 @@ class CorePing: TelemetryPing {
             out["defaultMailClient"] = chosenEmailClient as AnyObject?
         }
 
+        if #available(iOS 11.0, *) {
+            if let strength = self.prefs.stringForKey(ContentBlockerHelper.PrefKeyStrength), let enabled = self.prefs.stringForKey(ContentBlockerHelper.PrefKeyEnabledState) {
```

Comment:
The settings will only be reported if both have been set. You can probably split this up.

Reply:
oops! 👍